package.json

```diff
@@ -17,7 +17,7 @@
         "postrelease": "git push && git push --follow-tags && npm run build && npm publish",
         "test": "jest --env=jsdom src",
         "test:bundle": "jest --env=jsdom e2e-tests/bundle/**",
-        "test:e2e": "jest -c e2e-tests/jest.config.js",
+        "test:e2e": "PORT=4444 jest -c e2e-tests/jest.config.js",
```

Comment:
Should the port be removed here so users can set?

Reply:
This has been updated to the following:
```
"test:e2e": "BASE_URL=http://localhost:4444 jest -c e2e-tests/jest.config.js",
```

It is missing the ability for users to be able to override. But I don't think anyone ever used that feature. Let's drop it for simplicity. The real e2e tests live here and are much more configurable: https://github.com/paypal/paypal-sdk-e2e-tests